src/run.rs

```diff
@@ -187,7 +186,12 @@ pub fn run() {
     .collect::<Vec<&str>>();
 
   let justfile_option = matches.value_of("JUSTFILE");
-  let working_directory_option = matches.value_of("WORKING-DIRECTORY");
+  let mut working_directory_option = matches.value_of("WORKING-DIRECTORY");
+
+  if justfile_option.is_some() && working_directory_option.is_none() {
```

Comment:
Let's change this to an `if let`, so we can avoid the unwraps:

`if let (Some(justfile), None) = (justfile_option, working_directory_option) {`

Reply:
Done.